### PR TITLE
Implement CB's css fixes

### PR DESCRIPTION
**Pull request by bellesea**
https://docs.google.com/document/d/1Nih-PHROuxHbYljRDc-tRytS6qj92d07MzKnhaJZjas/edit

Hopefully it works? Fingers crossed haha. 

Thank you!

### CODE REVIEW

File: themes/hello-friend-ng/assets/scss/_locale.scss [hunk 1]
```diff
@@ -72,7 +76,6 @@
       }
 
       .locale {
-        padding-right: 15px;
         min-width: 0;
       }
   }
```
Comment (xanimo) [inline themes/hello-friend-ng/assets/scss/_locale.scss:81]: needs new line

File: themes/hello-friend-ng/assets/scss/_community.scss [hunk 2]
```diff
@@ -32,4 +45,22 @@
...
+  .doge-graphics {
+    position: relative;
+    width: 100%;
+    margin: auto;
+  }
+
+  .doge-graphics-letter {
+    margin: auto;
+  }
 }
```
Comment (xanimo) [inline themes/hello-friend-ng/assets/scss/_community.scss:66]: needs new line

File: themes/hello-friend-ng/assets/scss/_resources.scss [hunk 3]
```diff
@@ -118,3 +118,23 @@
...
+    padding: 1rem;
+    padding-bottom: 0.7rem;
+
+    a {
+      padding-bottom: 0.3rem;
+      text-decoration: none !important;
+      border-bottom: 1px solid #000000;
+    }
+  }
+}
```
Comment (xanimo) [inline themes/hello-friend-ng/assets/scss/_resources.scss:140]: needs new line

File: themes/hello-friend-ng/assets/scss/_wallets.scss [hunk 4]
```diff
@@ -33,3 +34,26 @@
...
+
+.google-play-button {
+  width: 15vw;
+  height: auto;
+  margin-top: 10px;
+  margin-left: -5px;
+}
+
+.wallet-bottom-container {
+  margin-top: 10vh;
```
Comment (xanimo) [inline themes/hello-friend-ng/assets/scss/_wallets.scss:58]: needs new line

File: themes/hello-friend-ng/assets/scss/_whatisdogecoin.scss [hunk 5]
```diff
@@ -344,3 +344,10 @@
     text-align: left;
   }
 }
+
+.gold-highlight {
+  color: #c4b367 !important;
+  padding: 3px 5px;
+  background-color: #faf5df;
+  border-radius: 1px;
+}
```
Comment (xanimo) [inline themes/hello-friend-ng/assets/scss/_whatisdogecoin.scss:353]: needs new line

**Review by xanimo — APPROVED** [review]
just some minor nits, new lines at end of some .scss files but other than that looks good but untested. utACK

**Comment by bellesea** [comment]
> just some minor nits, new lines at end of some .scss files but other than that looks good but untested. utACK

thank you! will make those changes :)) do you have any advice on the easiest way to test it? would it be to deploy this code somewhere and link it?

**Comment by xanimo** [comment]
> > just some minor nits, new lines at end of some .scss files but other than that looks good but untested. utACK
> 
> thank you! will make those changes :)) do you have any advice on the easiest way to test it? would it be to deploy this code somewhere and link it?

hey! yes were you not able to run on your local development environment? if so it may be that you need to install nodejs and npm and then run `npm i` in order to get dependencies prior to running hugo server. lmk if you have any issues.

**Comment by bellesea** [comment]
> > > just some minor nits, new lines at end of some .scss files but other than that looks good but untested. utACK
> > 
> > 
> > thank you! will make those changes :)) do you have any advice on the easiest way to test it? would it be to deploy this code somewhere and link it?
> 
> hey! yes were you not able to run on your local development environment? if so it may be that you need to install nodejs and npm and then run `npm i` in order to get dependencies prior to running hugo server. lmk if you have any issues.

@xanimo yes, i did run on my local dev environment while working on this :) do you have any suggestions on the best way to deploy it so that y'all can see what it looks like too? thank you!

**Comment by xanimo** [comment]
> > > > just some minor nits, new lines at end of some .scss files but other than that looks good but untested. utACK
> > > 
> > > 
> > > thank you! will make those changes :)) do you have any advice on the easiest way to test it? would it be to deploy this code somewhere and link it?
> > 
> > 
> > hey! yes were you not able to run on your local development environment? if so it may be that you need to install nodejs and npm and then run `npm i` in order to get dependencies prior to running hugo server. lmk if you have any issues.
> 
> @xanimo yes, i did run on my local dev environment while working on this :) do you have any suggestions on the best way to deploy it so that y'all can see what it looks like too? thank you!

yeah so without doing a deep dive it should be possible to configure deploying to gh pages on each push which would be shareable via bellesea.github.io/dogecoin.com or whatever url you desire. we can do a walk through together if you'd like to pursue that but in the meantime i've finally built and gone over everything for the most part while using cb's google doc as reference.

everything looks p good but have some minor changes again. will post them here and if you can't get to it will merge and create new issues for somebody else:

1) the background here is getting cut off
Screenshot from 2022-04-25 14-01-17
which can be solved by removing the background-position from .wallets-bg class and then adding padding to top of .wallet class:
Screenshot from 2022-04-25 14-02-25

2) the .wallet-bottom-container could use some padding to top as show here:
Screenshot from 2022-04-25 14-05-06

3) (optional) this one is contrary to cb's request, i personally think it looks weird having padding-left with no padding on the right but that's just me. asking for your opinion:

Screenshot from 2022-04-25 14-07-26

my preference:
Screenshot from 2022-04-25 14-07-00

maybe we can all agree to meet in the middle? haha ;)
Screenshot from 2022-04-25 14-07-26
 
4) last is padding-top for your tablist (targetting col-lg-3... and col-lg-1...):
Screenshot from 2022-04-25 14-18-44

and i suppose cb wants the dogepedia img adjacent to the tablist larger so if possible to make it a bit bigger would be good.

regardless this all looks great and i thank you for your time and awesome work! hmu on discord if you'd like to pursue deploying preview on your github account and/or lmk if this is doable otherwise will convert these to issues.

**Comment by bellesea** [comment]
@xanimo thanks for the comments, just pushed all the changes except for the DOGE one... i don't have strong opinions but i do kinda like that it's different from other elements on the page! maybe CB can chime in on this?

on deploying w/ github, I'll try and figure it out this weekend for future dev work hehe :) thank you for the offer, will reach out if i need help!

**Review by exu3 — APPROVED** [review]
Sorry about the late review; I think this looks great. Nice work Belle!

**Comment by qlpqlp** [comment]
I will close this PR because there was significante changes that now have to be considered :)

**Pull request closed without merging**